template.php

```diff
@@ -133,6 +133,29 @@ function chs_theme_preprocess_islandora_solr_metadata_display(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess().
+ */
+function chs_theme_process_islandora_solr_metadata_display(&$variables) {
+  $solr_fields = $variables['solr_fields'];
+
+  if (isset($solr_fields['mods_accessCondition_useAndReproduction_xlinkhref_ms']) && isset($solr_fields['mods_accessCondition_useAndReproduction_ms'])) {
```

Comment:
I wouldn't expose having the `mods_accessCondition_useAndReproduction_xlinkhref_ms` field being present in the metadata display as it clogs up the UI when it doesn't have to. Seeing as you are grabbing the value and unsetting it later you could add it to `$variables['solr_fields']` array in https://github.com/discoverygarden/chs_theme/blob/7.x/template.php#L128-Lundefined and unsetting it here again after using it.